pkg/scaffold/ansibleterraform/playbook_app_yml.go

```diff
@@ -27,7 +27,7 @@ func (t *ApplicationYml) GetInput() (input.Input, error) {
 
 const applicationYmlTmpl = `---
 - name: converge the application roles
-  hosts: {{.AppName}}
+  hosts: {{.AppName | replace "-" "_" }}
```

Comment:
ansibe considers inventory groups with dashes in their names as invalid characters so we change occurrences of dash characters to underscores.